app/api/events.py

```diff
@@ -168,7 +168,7 @@ def create(self, data, url):
         save_to_db(new_event, "Event saved")
         # set organizer
         role = Role.query.filter_by(name=ORGANIZER).first()
-        uer = UsersEventsRoles(g.user, new_event, role)
+        uer = UsersEventsRoles(user_id=creator_id, event=new_event, role=role)
```

Comment:
I haven't pushed it yet. I am asking if this is a good way? We don't have to pass creator_id for zip imports.